.gitignore

```diff
@@ -44,3 +44,4 @@ imports/plugins/custom/*
 !imports/plugins/custom/.gitkeep
 
 .reaction/config.json
+package-lock.json
```

Comment:
New line please.
You have decided to use npm instead of yarn? Just clarifying why `package-lock.json` is ignored